### ISSUE TITLE
diff-highlight: command not found

### ISSUE THREAD

**Issue by tleunen** [so-fancy/diff-so-fancy#164]
I recently updated my node version and for some reasons, diff-so-fancy doesn't work anymore.

I even tried to uninstall/reinstall it with brew, no success.
```
~/d/project master ❯ brew install diff-so-fancy
==> Downloading https://github.com/so-fancy/diff-so-fancy/archive/v0.9.3.tar.gz
Already downloaded: /Users/tommy.leunen/Library/Caches/Homebrew/diff-so-fancy-0.9.3.tar.gz
🍺  /usr/local/Cellar/diff-so-fancy/0.9.3: 8 files, 32.4K, built in 0 seconds

~/d/project master ❯ git log
diff-highlight | diff-so-fancy | less -r: diff-highlight: command not found
```

**Comment by stevemao** [comment]
Can you make sure if `diff-so-fancy` is in your `$PATH`?

**Comment by tleunen** [comment]
Yes `diff-so-fancy` is in the path, but `diff-highlight` is not

**Comment by paulirish** [comment]
you should update your git pager config. You shouldn't need highlight in the config any longer. dsf does it automatically.

**Comment by tleunen** [comment]
Thanks.
By removing diff-highlight from the config fixes the issue.

**Comment by mvcds** [comment]
In case you (who got here through Google) have a similar issue and you're using something like `nvm` check if your current node version has it installed.

**Comment by waldyrious** [comment]
Thanks, @mvcds, that set me in the right path. It was indeed a package manager versioning issue, though not exactly `nvm`. In my case I had previously made `diff-highlight` available in the PATH via this symlink:

    /usr/local/bin/diff-highlight -> /usr/local/Cellar/git/2.19.1/share/git-core/contrib/diff-highlight/diff-highlight

...and the link became broken when I updated git to 2.20 using brew, and later ran `brew cleanup` which deleted git's 2.19 folder in Homebrew's Cellar.

To prevent a similar situation in the future should I update git again, I changed the link to

    /usr/local/bin/diff-highlight -> /usr/local/opt/git/share/git-core/contrib/diff-highlight/diff-highlight

...having gotten the `/usr/local/opt/git` path using `brew --prefix git`.
